bigtop-packages/src/common/ignite-hadoop/do-component-build

```diff
@@ -21,4 +21,23 @@ set -ex
 export MAVEN_OPTS="-Xmx512M"
 
 sed -i '/asm-all/{n;s/4.2/5.0.2/}' modules/hadoop/pom.xml
+
+# download ignite-shmem from maven central
+mvn dependency:get -DremoteRepositories=https://repo1.maven.org/maven2/ -DgroupId=org.gridgain -DartifactId=ignite-shmem -Dversion=1.0.0
+
+cd ipc/shmem
+# patch and build it
+sed -i 's/\&ipcData->closed\,/(char *)\&ipcData->closed\,/' igniteshmem/org_apache_ignite_internal_util_ipc_shmem_IpcSharedMemoryUtils.cpp
+./configure && make
+# unpack
+jar xf ~/.m2/repository/org/gridgain/ignite-shmem/1.0.0/ignite-shmem-1.0.0.jar
+# replace
+cp ./igniteshmem/.libs/libigniteshmem.so META-INF/native/linux64/libigniteshmem.so
+# repack
+jar cf ignite-shmem-1.0.0.jar META-INF
+cp ignite-shmem-1.0.0.jar ~/.m2/repository/org/gridgain/ignite-shmem/1.0.0/ignite-shmem-1.0.0.jar
```

Comment:
Please use `mvn install:install-file -Dfile=target/dependency/ignite-shmem-1.0.0.jar -DgroupId=org.gridgain -DartifactId=ignite-shmem -Dversion=1.0.0    -Dpackaging=jar -DcreateChecksum=true` instead.

It will create correct checksums in maven repo as well.